src/shared/getSpeakerTime.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+
+interface Speaker {
+  name: string;
+  totalSpeakingSeconds: number;
+}
+
+export function getSpeakerTime(filePath: string): Promise<Speaker[]> {
```

Comment:
1. pass in the transcript string directly instead of a file. we aren't going to store transcripts as files.
2. without file reading it won't need to return Promise any more.